more/free-programming-cheatsheets.md

```diff
@@ -85,6 +85,7 @@
 
 ### Artificial Intelligence
 
+* [AI Terminology Simplified: Key Concepts Explained for Humans](https://www.theverge.com/24201441/ai-terminology-explained-humans) TheVerge (HTML)
```

Comment:
Where did you see this title? I see "AI is confusing — here’s your cheat sheet". I also see an author credit to Jay Peters. I would credit him rather than The Verge.

Reply:
I initially thought of changing the title inorder to sum up what is there in this article, but I agree that we can keep is "AI is confusing — here’s your cheat sheet" ,will do the changes!